packages/tocco-ui/src/FormattedValue/typeFormatters/PhoneFormatter.js

```diff
@@ -16,9 +16,11 @@ class PhoneFormatter extends React.Component {
 
   render() {
     const libPhoneImport = this.state.libPhoneImport
-    const parsed = libPhoneImport ? libPhoneImport.parseNumber(this.props.value) : this.props.value
-    const formattedNumber = libPhoneImport ? libPhoneImport.formatNumber(parsed, 'International') : parsed
-    return (<span>{_isEmpty(parsed) ? parsed : formattedNumber}</span>)
+    const parsed = libPhoneImport ? libPhoneImport.parseNumber(this.props.value, {extended: true}) : this.props.value
+    const validityParsed = parsed.valid ? parsed : this.props.value
+    const notEmptyValue = _isEmpty(validityParsed) ? this.props.value : validityParsed
+    const formattedNumber = libPhoneImport ? libPhoneImport.formatNumber(notEmptyValue, 'International') : notEmptyValue
+    return (<span>{formattedNumber}</span>)
```

Comment:
You check 2 times if libPhoneImport is defined. Does it have to be that complicated? Would the old version work with a simple check?

something like:
```
if (libPhoneImport) {
const parsed = libPhoneImport.parseNumber(props.value)
return <span>{_isEmpty(parsed) ? props.value : libPhoneImport.formatNumber(parsed, 'International')}</span>
} ellse {
 return <span>{props.value}</span>
}
```

you can also move it to a function

`return <span>{libPhoneImport ? formatWithLib(...) : props.value}</span>`